app/dashboard/dashboard.tsx

```diff
@@ -26,12 +34,18 @@ export const links = () => {
   ];
 };
 
+export const Avatar = styled("img", {
+  width: "$5",
+  borderRadius: "50%",
+});
+
 type DashboardProps = {
   projects?: Array<{ id: string; title: string }>;
+  user?: User;
```

Comment:
at this point user can't be optional or can it?

Reply:
I mean if there is no user we shouldn't be rendering dashboard

Reply:
Mistake, fixed!